application/src/main/java/org/togetherjava/tjbot/commands/tophelper/TopHelpersMessageListener.java

```diff
@@ -18,6 +18,8 @@
  * {@link TopHelpersCommand} to pick them up.
  */
 public final class TopHelpersMessageListener extends MessageReceiverAdapter {
+    private static final String UNCOUNTED_CHARS = "[^\\x20-\\x7E]";
```

Comment:
that regex is ugly and unmaintainable. there are a few better alternatives. first, a direct translation that is at least readable `[^ -~]` (space to tilde).

but I would actually go for an approach using the named classes, such as `\P{C}`, which are all visible characters.

More here: https://stackoverflow.com/a/66340317/2411243

Also, such unreadable code at least needs a quick comment explaining it (and a link to the SO where u got it from, if any).

Reply:
That said, I would also be okay with just `\\W` (non word characters).

Reply:
For math and algorithm-based help there would be a significant number of non-word characters.

Reply:
hm i see that is way more readable